### DISCUSSION TITLE
HSR Stops responding after collision

### DISCUSSION THREAD

**Discussion by Kai-X-Org** [HumanSupportRobot/QandA#75] [Q&A]
For our simulations, we realized that the HSR would stop responding to commands would give the following error:
---------------------------------------------------------------------------
MobileBaseError                           Traceback (most recent call last)
<ipython-input-5-acefd79a5298> in <module>
----> 1 sim.base.go_rel(-1.0)

/opt/ros/noetic/lib/python3/dist-packages/hsrb_interface/mobile_base.py in go_rel(self, x, y, yaw, timeout)
    121         ref_frame_id = settings.get_frame('base')
    122         goal = self.create_go_pose_goal(pose, ref_frame_id)
--> 123         self._send_goal_pose_and_wait(goal, timeout)
    124 
    125     def go_abs(self, x=0.0, y=0.0, yaw=0.0, timeout=0.0):

/opt/ros/noetic/lib/python3/dist-packages/hsrb_interface/mobile_base.py in _send_goal_pose_and_wait(self, goal, timeout)
    199                     error_text = self._action_client.get_goal_status_text()
    200                     msg = 'Failed to reach goal ({0})'.format(error_text)
--> 201                     raise exceptions.MobileBaseError(msg)
    202             else:
    203                 self._action_client.cancel_goal()

MobileBaseError: Failed to reach goal ()

Is ther a fix to this without restarting the simulator? Thanks!

**Comment by masudam** [comment]
The function "base.go_rel" is used to move the base relatively,  but in reality, it calculates the absolute target position internally and sends commands to the base's action server. The error message indicates that the base did not reach the goal, but it doesn't specify the reason for the failure.

Possible reasons could be the presence of obstacles at the goal location, getting stuck due to steps or other obstacles, among others.

As additional information, in the "megaweb_2015_world" environment, there is a step at the entrance of the room located 1.0m below the launch's initial position, so there is a possibility of getting stuck there.
